recipes/fenics/build.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+
+if [[ "$(uname)" == "Darwin" ]]; then
+  export MACOSX_DEPLOYMENT_TARGET=10.9
+  export CXXFLAGS="-std=c++11 -stdlib=libc++ $CXXFLAGS"
+fi
+
+for pkg in dijitso ufl instant fiat ffc; do
+    echo "installing ${pkg}-${PKG_VERSION}"
+    git clone -q --depth 1 -b ${pkg}-${PKG_VERSION} https://bitbucket.org/fenics-project/${pkg}.git
+    pushd $pkg
+    pip install --no-deps .
+    popd
+done
+
+pkg=dolfin
+echo "installing ${pkg}-${PKG_VERSION}"
+git clone -q --depth 1 -b ${pkg}-${PKG_VERSION} https://bitbucket.org/fenics-project/${pkg}.git
```

Comment:
Good call, I'll add a note. I did just figure out a way to make the 'main' package (dolfin) the source in the meta.yaml, so it will at least be a little less weird.